sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_content_safety/__init__.py

```diff
@@ -16,6 +15,5 @@
     "SelfHarmEvaluator",
     "HateUnfairnessEvaluator",
     "ContentSafetyEvaluator",
-    "ContentSafetyChatEvaluator",
     "ContentSafetyEvaluatorBase",
```

Comment:
I think we retire this class now, right?

Reply:
Yes I removed the ContentSafetyChatEvaluator and ContentSafetyEvaluatorBase. Or is there something extra I need to do to retire a class?